aspnet/migration/aspnetcoremigration.rst

```diff
@@ -0,0 +1,190 @@
+Migrating from ASP.NET to ASP.NET Core
```

Comment:
This doc is about migrating from ASP.NET 5 RC1 to ASP.NET Core. It's not about migrating from ASP.NET (like your Web Forms app) to ASP.NET Core.